CHANGELOG.md

```diff
@@ -1,10 +1,15 @@
+## v5.2.0 (October 18, 2019)
```

Comment:
We should be using semantic-versioning to handle this, remove v5.2.0 additions in the CHANGELOG.md and rename commit to `feat: adding debug grid + grid mixins`, see guide below

![image](https://user-images.githubusercontent.com/5377775/67337826-ccb3a180-f4f5-11e9-98a2-e9cfc4dc9ed2.png)

`npm run release` will auto update the changelog based on those commit keys (like `feat:, fix:, etc.` and update the tag and the package version

Reply:
good point @ragu-S 
I'll update